homeassistant/components/home_connect/icons.json

```diff
@@ -23,43 +23,144 @@
     }
   },
   "entity": {
+    "binary_sensor": {
+      "remote_control": {
+        "default": "mdi:remote",
+        "state": {
+          "on": "mdi:remote",
+          "off": "mdi:remote-off"
+        }
+      },
+      "remote_start": {
+        "default": "mdi:remote",
+        "state": {
+          "on": "mdi:remote",
+          "off": "mdi:remote-off"
+        }
```

Comment:
If you use a default, you can omit the state where you use the same as the default